corehq/apps/hqwebapp/views.py

```diff
@@ -529,32 +513,63 @@ def bug_report(req):
     report['full_name'] = full_name
     report['email'] = email or report['username']
 
-    matching_subscriptions = Subscription.objects.filter(
-        is_active=True,
-        subscriber__domain=report['domain'],
-    )
-
-    if len(matching_subscriptions) >= 1:
-        report['software_plan'] = matching_subscriptions[0].plan_version
+    if report['domain']:
+        domain = report['domain']
+    elif len(couch_user.domains) == 1:
+        # This isn't a domain page, but the user has only one domain, so let's use that
+        domain = couch_user.domains[0]
```

Comment:
Cool, I hadn't seen that one.  I don't think it'll be available on non-domain pages, though, since it's underneath a `if 'domain' in view_kwargs:` check.